gwr/gwr.py

```diff
@@ -888,6 +890,74 @@ def adj_pseudoR2(self):
     def pvalues(self):
         raise NotImplementedError('Not implemented for GWR')
 
+    @cache_readonly
+    def local_collinearity(self):
+        """
+        Computes several indicators of multicollinearity within a geographically
+        weighted design matrix, including:
+        
+        local correlation coefficients (n, ((p**2) + p) / 2)
+        local variance inflation factors (VIF) (n, p-1)
+        local condition number (n, 1)
```

Comment:
Just a minor thing, but `local_CN` returned is in shape of (n, )

Reply:
Cool, fixed now.